test/java/seedu/addressbook/commands/SortCommandTest.java

```diff
@@ -0,0 +1,59 @@
+package seedu.addressbook.commands;
```

Comment:
Good job adding JUnit test for the new feature 👍 . It's better to add/modify tests for other relevant classes also (i.e. `parser`)